hw2/src/main/resources/json/FalconGanonJson.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
I think you can put a root key after the curly brace , like the name represent what is inside it.